### ISSUE TITLE
Recurring transactions get group title on overview page

### ISSUE THREAD

**Issue by zeitwidrig** [firefly-iii/firefly-iii#9305]
### Support guidelines

- [x] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [x] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem
- [x] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

Recurring transactions are getting a group title (as in split transactions) which is displayed on the overview page.

![Image](https://github.com/user-attachments/assets/1388355b-d082-4e0a-bdc2-a7d48aeefa15)

I'm pretty sure it is related to this commit https://github.com/firefly-iii/firefly-iii/commit/84a1a876e1dcef0a38718f79c23e3f9b373998f3 which fixed #8844 but introduced this little regression.

### Debug information

Debug information generated at 2024-10-03 23:33:16 for Firefly III version **v6.1.21**.

<table><thead><tr><th colspan="2">System information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Firefly III</td><td>6.1.21 / v2.1.0 / 24 (exp. 24)</td></tr><tr><td>PHP version</td><td>8.3.8 (64bits) / fpm-fcgi / Linux x86_64</td></tr><tr><td>BCscale</td><td>12</td></tr><tr><td>Error reporting</td><td>Display: Off, reporting: ALL errors</td></tr><tr><td>Max upload</td><td>2147483648 (2147483648 bytes)</td></tr><tr><td>Database drivers</td><td><strong>*mysql*</strong>, pgsql, sqlite, </td></tr></tbody></table><table><thead><tr><th colspan="2">Firefly III information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>Timezone</td><td>Europe/Amsterdam + Europe/Berlin</td></tr><tr><td>App environment</td><td>local, debug: false</td></tr><tr><td>Layout</td><td>v1</td></tr><tr><td>Logging</td><td>notice, daily / (empty)</td></tr><tr><td>Cache driver</td><td>file</td></tr><tr><td>Default language and locale</td><td>en_US + equal</td></tr><tr><td>Trusted proxies</td><td></td></tr><tr><td>Login provider & user guard</td><td>eloquent / web</td></tr><tr><td>Login headers</td><td>N/A + N/A</td></tr><tr><td>Stateful domains</td><td></td></tr><tr><td>Last cron job</td><td>2024-10-03 01:00:20 (20 hours ago)</td></tr><tr><td>Mailer</td><td>log</td></tr></tbody></table><table><thead><tr><th colspan="2">User-specific information</th></tr><tr><th>Item</th><th>Value</th></tr></thead><tbody><tr><td>User</td><td><span>#</span>1 of 1</td></tr><tr><td>User flags</td><td><span title="Has rules">:wrench:</span> <span title="Has recurring transactions">:clock130:</span> <span title="Has object groups">:bookmark_tabs:</span> <span title="Has subscriptions">:email:</span></td></tr><tr><td>Session start</td><td>2024-10-01 22:56:18</td></tr><tr><td>Session end</td><td>2024-10-31 22:56:18</td></tr><tr><td>View range</td><td>1M</td></tr><tr><td>User language</td><td>en_GB</td></tr><tr><td>User locale</td><td>en_GB</td></tr><tr><td>Locale(s) supported</td><td>en_GB.utf8: :white_check_mark:<br>en_GB.UTF-8: :white_check_mark:<br></td></tr><tr><td>User agent</td><td>Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:130.0) Gecko/20100101 Firefox/130.0</td></tr></tbody></table>

### Expected behaviour

Recurring transactions shouldn't get a group title (just as all other single transactions)
So basically the behaviour before the commit.

### Steps to reproduce

1. create new recurring transaction
2. trigger future transaction via "create right now"
3. check on overview page

Can be reproduced on the demo site:

![Image](https://github.com/user-attachments/assets/5ea360ff-2f3a-4d2b-85a5-8ee0e50c4102)

**Comment by JC5** [comment]
Nice, I'll pick it up! Thanks for opening an issue.

**Comment by JC5** [comment]
Fixed in 6.1.22
